### PR TITLE
Adding Brazil time zones

### PR DESCRIPTION
**Pull request by DmitryGrayscale**
According to the following article (https://en.wikipedia.org/wiki/Time_in_Brazil), there are 6 possible regions in Brazil with their own time rules, so I've added a time zone per each region and named them after states (in the same way as it was done for Australia I guess).
Though I don't know how it should look like exactly, _time zones are big mess_

What do you think?

P.S. Also noticed that there are no abbrevs (like CEST, EDT, MSK etc) for Brazilian time zones, arrow just returns  "-03" and I think it's going to look a bit weird in resulting embed, so I tried to workaround it.

### CODE REVIEW

File: apollo/embeds/event_embed.py [hunk 1]
```diff
@@ -81,6 +81,16 @@ def _standby_members(self, responses, guild, event_capacity):
...
         start_time = event.local_start_time.format("ddd MMM Do, YYYY @ h:mm A")
         time_zone = event.local_start_time.tzname()
+
+        # sometimes there's no adequate abbreviation for tz
+        if any(char.isdigit() for char in time_zone):
+            if len(time_zone) == 3:
+                # will look like 'UTC+03' for example
+                time_zone = "UTC" + time_zone
+            else:
+                # there are tznames like '+0120', '0830' etc. Just don't show them
```
Comment (jgayfer) [inline apollo/embeds/event_embed.py]: Omit the timezone? How would we know when the event is?

**Comment by DmitryGrayscale** [comment]
It doesn't want me to trigger CI it seems

**Comment by jgayfer** [comment]
Don’t worry about CI, I don’t have it set up to build PRs properly :)

File: locales/en.yml [hunk 2]
```diff
@@ -122,6 +122,12 @@ time_zones:
   america/mexico_city: Central Time (US & Canada)
   america/detroit: Eastern Time (US & Canada)
   america/phoenix: Arizona Time (US)
+  america/rio_branco: Acre (Brasil)
+  america/manaus: Amazonas (Brazil)
+  america/cuiaba: Mato Grosso (Brazil)
+  america/bahia: Bahia (Brasil)
+  america/sao_paulo: Sao Paulo (Brasil)
+  america/noronha: Fernando de Noronha (Brasil)
```
Comment (jgayfer) [inline locales/en.yml]: Some of these spell it `Brazil` and some `Brasil` - Is this intentional?

Comment (jgayfer) [reply locales/en.yml]: Noticed while testing it out.

Comment (DmitryGrayscale) [reply locales/en.yml]: just a typo :)

**Comment by jgayfer** [comment]
Merged! Thank you very much.

**Pull request merged**